lib/main/scrapper.js

```diff
@@ -1,22 +1,23 @@
 var cheerio = require('cheerio');
 var http = require('http');
 var url = require('url');
+var RSVP = require('RSVP');
 
-function Scrapper(uri, success_callback, error_callback){
-  if(! /^http:/.test(uri) ){
-    uri = "http://" + uri;
-  }
-  http.get(uri, function(response){
-    var body = '';
-    response.on('data', function(d) {
-      body += d;
+function Scrapper(uri){
+  if(! /^http:/.test(uri)){ uri = "http://" + uri; }
+  return new RSVP.Promise(function(fullfill, reject){
```

Comment:
It is our norm to use official terminology of a library to avoid confusion. e.g. we will use 'resolve' instead of 'fullfill' to avoid confusion